### PR TITLE
feat(client): adds creation and saving project features

### PR DESCRIPTION
**Pull request by jkergal**
## Ticket

**Title: ETQM, je peux enregistrer mes projets de code**
**Notion Link: https://www.notion.so/jkergal/ETQM-je-peux-enregistrer-mes-projets-de-code-8a03b662779c4fd0a8482c6e0036043d?pvs=4**

## Description
- adds a project management bar on top of the page
- adds an input to update the project name
- adds a dynmic button to save project
- plugs to back-end

## Guidelines

🔗 Link your PRs to their related Notion tickets
✍️ Use the commit convention to your pull request title
📃 If your code affects the build, update `README.md`
🚫 Do not merge a PR until all comments are resolved
🗑 Remove your branch after merging

image
image

### CODE REVIEW

File: server/src/seeds/project.seeds.ts [hunk 1]
```diff
@@ -1,42 +1,46 @@
...
-		{
-			path: '/constants/color.ts',
-			content: "export const BLACK = 'black'",
-			projectId: project.id,
-			isHidden: false,
-			programmingLanguage: ProgrammingLanguages.TYPESCRIPT
-		}
-	];
-};
+// export const generateFileFixtures = (project: Project) => {
```
Comment (reynan-dev) [inline server/src/seeds/project.seeds.ts:26]: Why not remove this comments?

File: server/src/seeds/project.seeds.ts [hunk 2]
```diff
@@ -53,17 +57,17 @@ export const createProjects = async () => {
 			.values(projectFixture)
 			.execute();
 
-		if (!project) return console.error('Project is needed to create files');
+		// if (!project) return console.error('Project is needed to create files');
```
Comment (reynan-dev) [inline server/src/seeds/project.seeds.ts:60]: Same for this comments

File: server/src/utils/methods/CustomAuthChecker.ts [hunk 3]
```diff
@@ -0,0 +1,19 @@
+// import { Context } from "apollo-server-core";
```
Comment (reynan-dev) [inline server/src/utils/methods/CustomAuthChecker.ts:1]: Same here..

**Review by reynan-dev — APPROVED** [review]
Remove the comments

File: server/src/utils/methods/CustomAuthChecker.ts (hunk 3 above)
Comment (jkergal) [reply server/src/utils/methods/CustomAuthChecker.ts:1]: > Remove the comments

@reynan-dev OMG so sorry about that... I'll remove it

**Pull request merged**